thinkhazard/scripts/s3helper.py

```diff
@@ -1,20 +1,24 @@
 import boto3
 import logging
+import os
 from botocore.client import Config
 from botocore.exceptions import ClientError
 
 
 class S3Helper:
-    def __init__(self, bucket, **kwargs):
-        # only set endpoint_url for debug, boto generates aws endpoint_url for prod
-        if kwargs["aws_access_key_id"] == "minioadmin":
-            kwargs["endpoint_url"] = "http://minio:9000/"
+    def __init__(self, settings):
+        # only set endpoint_url for dev, boto generates aws endpoint_url for prod
+        kwargs = ({"endpoint_url": "http://minio:9000/"}
+                  if os.environ['INI_FILE'] == "c2c://development.ini"
+                  else {})
```

Comment:
This is not a strong strategy.
We might use development.ini on OpenShift or production.ini locally.
I would recommend an AWS_ENDPOINT_URL env var.

Reply:
True, adapted accordingly. Thanks